docs/source/api_index.rst

```diff
@@ -95,8 +95,8 @@ Pipeline Base Classes
 .. autoapisummary::
 
     evalml.pipelines.PipelineBase
-    evalml.pipelines.ClassificationPipeline
-    evalml.pipelines.BinaryClassificationPipeline
+    evalml.pipelines.classification_pipeline.ClassificationPipeline
```

Comment:
Docs break without this update (see [here](https://readthedocs.com/projects/feature-labs-inc-evalml/builds/683782/)); filed an issue to track this long term as #2548